File: E-CommerceMobileApp/app/src/main/java/com/technoio/models/Categorie.java
package com.technoio.models;


import java.util.List;

public class Categorie {

    public int ID;
    public String Nom;
    public List<Produit> produitList;

    public Categorie(int ID, String nom) {
        this.ID = ID;
        this.Nom = nom;
    }

}

File: E-CommerceFullStack/E-CommerceApi/Models/Categorie.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace E_CommerceApi.Models
{
    public class Categorie
    {
        public int ID { get; set; }
        public string Nom { get; set; }

        public List<Produit> Produits { get; set; }
    }

}
